pkg/cmd/api/api_test.go

```diff
@@ -1355,10 +1365,10 @@ func Test_apiRun_cache(t *testing.T) {
 	ios, _, stdout, stderr := iostreams.Test()
 	options := ApiOptions{
 		IO: ios,
-		Config: func() (gh.Config, error) {
+		Config: func() (cfg gh.Config, err error) {
 			return &ghmock.ConfigMock{
 				AuthenticationFunc: func() gh.AuthConfig {
-					return &config.AuthConfig{}
+					return &stubAuthConfig{}
```

Comment:
Since `cfg` isn't exported, I couldn't do something like `&config.AuthConfig{cfg: cfg}` which otherwise probably makes sense. Considered exporting `Cfg` but it seemed too invasive so used a stub pattern I found in another test instead.